### PR TITLE
added pull request template  and issue template to root folder

### PR DESCRIPTION
**Pull request by wordhouse66**
I added a PR Template in the form of pull_request_template.md to the root folder. 

This should help address and fix issue #71

I also added a Issue Template in the form of issue_template.md to the root folder. 

This should help address and fix issue #70

### CODE REVIEW

File: issue_template.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Please answer the following questions for yourself before submitting an issue. **YOU MAY DELETE THE PREREQUISITES SECTION.**
+
+- [ ] I am running the latest version
+- [ ] I checked the documentation and found no answer
+- [ ] I checked to make sure that this issue has not already been filed
+
+
+# Expected Behavior
+
+Please describe the behavior you are expecting
```
Comment (aonomike) [inline issue_template.md:12]: Could we use comments here instead?

File: issue_template.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+# Current Behavior
+
+What is the current behavior?
```
Comment (aonomike) [inline issue_template.md:16]: same here?

File: issue_template.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+# Failure Information (for bugs)
+
+Please help provide information about the failure if this is a bug. If it is not a bug, please remove the rest of this template.
```
Comment (aonomike) [inline issue_template.md:20]: and here :)?

File: issue_template.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+Please provide detailed steps for reproducing the issue.
+
+1. step 1
+2. step 2
+3. you get it...
+
+## Context
+
+Please provide any relevant information about your setup. This is important in case the issue is not reproducible except for under certain conditions.
```
Comment (aonomike) [inline issue_template.md:32]: and here?

**Review by aonomike — CHANGES_REQUESTED** [review]
Thanks Chima. As discussed, please use comments instead.

File: issue_template.md (hunk 1 above)
Comment (wordhouse66) [reply issue_template.md:12]: When you say 'INSTEAD', do you mean I should have  'Please describe the behavior you are expecting'  become  for instance ?

**Pull request closed without merging**